examples/federated-learning/tff/nvflare/requirements.txt

```diff
@@ -0,0 +1 @@
+nvflare==2.4.1
```

Comment:
Can you add this requirements file to dependabot.yml?

Reply:
Done